libraries/Filter/AP_Filter_config.h

```diff
@@ -3,15 +3,23 @@
 #include <AP_HAL/AP_HAL_Boards.h>
 
 #ifndef AP_FILTER_ENABLED
-#define AP_FILTER_ENABLED BOARD_FLASH_SIZE > 1024
+#define AP_FILTER_ENABLED HAL_PROGRAM_SIZE_LIMIT_KB > 1024
+#endif
+
+#ifndef AP_FILTER_NUM_FILTERS
+#if HAL_PROGRAM_SIZE_LIMIT_KB > 1024
+#define AP_FILTER_NUM_FILTERS 8
+#else
+#define AP_FILTER_NUM_FILTERS 0
```

Comment:
this breaks https://github.com/ArduPilot/ardupilot/pull/28787

Reply:
Fixed